### PR TITLE
Allow build team to list and view SA and list secrets

### PR DESCRIPTION
**Pull request by tkdchen**
[STONEBLD-1611](https://issues.redhat.com//browse/STONEBLD-1611)

Build team members are added to allow list and view ServiceAccount and list secrets across namespaces.

### CODE REVIEW

**Review by rcerven — APPROVED** [review]

File: components/build-service/base/rbac/kustomization.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 resources:
 - build-maintainer.yaml
+- build-team-can-view-sa-secret.yaml
```
Comment (mmorhun) [inline components/build-service/base/rbac/kustomization.yaml:3]: I think we cannot view secrets.

Comment (rcerven) [reply components/build-service/base/rbac/kustomization.yaml:3]: yeah but that doesn't help with debugging tbh

Comment (tkdchen) [reply components/build-service/base/rbac/kustomization.yaml:3]: only verb list is granted for Secret resource

**Review by Michkov — APPROVED** [review]

**Review by mmorhun — APPROVED** [review]

**Comment by tkdchen** [comment]
Conflict is resolved.

**Pull request merged**